src/styles/Blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+$main-color: #00acdc;
+$main-background-color: #fff;
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: flex;
+  height: 60px;
+  margin-right: 20px;
+  list-style: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  position: relative;
+
+  &:hover {
+    color: $main-color;
+  }
+}
+
+.is-active {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  position: relative;
+  color: $main-color;
+
+  &:after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    width: 37px;
```

Comment:
you don't need to hardcode values, just `width: 100%`

Reply:
+ fixed it